pkg/services/ngalert/schedule/schedule.go

```diff
@@ -395,35 +382,24 @@ func (sch *schedule) ruleRoutine(grafanaCtx context.Context, key ngmodels.AlertR
 	}
 
 	evalRunning := false
-	var currentRule *ngmodels.AlertRule
+	var currentRuleVersion int64 = math.MinInt64
```

Comment:
Do versions start at 0 or 1?

Reply:
It starts with 1. However, I do not feel confident using 0 because it is a default value for the type. As long as this variable is used for "greater than" comparison, I think the minimal possible value is the best solution.

Reply:
I think we should feel confident using the default value here. We can even add two if statements if that's better: 

1. If `lastVersion` is `0` then we should log an error because this is an invalid version
2. We don't need to document the significance of using `math.MinInt64` as the initial value for `currentRuleVersion`